modules/julia/src/cv.cpp

```diff
@@ -0,0 +1,14 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+// Copyright (C) 2020 by Archit Rungta
+
+// Needed to prevent documentation warning
```

Comment:
This is strange.
Doxygen should not look into "src" directory.

Reply:
Yes, it's only needed because this function is defined in the header file to prevent documentation warning.